style.css

```diff
@@ -91,3 +81,193 @@ h1 span {
 .page-links li {
   margin-right: 19px;
 }
+
+.my-projects {
+  padding: 15px;
+  margin: 20px;
+}
+
+.down-arrow {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 15px;
+}
+
+.title {
+  display: flex;
+  justify-content: center;
+  margin: 10px;
+}
+
+.line {
+  display: flex;
+  justify-content: center;
+  margin: 20px;
+}
+
+.project-container {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 20px;
+}
+
+.single-project {
+  height: 474px;
+  width: 327px;
+  border-radius: 8px;
+  border: 2px solid #ebf0ee;
+  margin: 5px auto;
+}
+
+.single-project img {
+  width: 100%;
+  height: 55%;
+}
+
+.single-project .project-title {
+  display: flex;
+  justify-content: center;
+  margin: 12px;
+}
+
+.single-project .project-title h4 {
+  text-align: center;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 30px;
+  color: #3a4a42;
+}
+
+.tech-stack {
+  display: flex;
+  justify-content: space-around;
+  margin: 15px 0;
+}
+
+.tech-stack li {
+  padding: 8px;
+  border-radius: 4px;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 600;
+  background: #ebf0ee;
+  color: #3a4a42;
+}
+
+.btn {
+  display: flex;
+  justify-content: center;
+  margin: 22px;
+}
+
+.btn button {
+  background: #36b37f;
+  font-weight: 500;
+  border-radius: 8px;
+  border: none;
+  color: #fff;
+  padding: 15px 20px;
+  outline: none;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.btn button:hover {
+  background-color: rgb(43, 107, 43);
+}
```

Comment:
- [ ] [REQUIRED] Thank you for implementing the hover interaction on your button, however, you have not implemented the enabled and pressed interaction. Please implement this.
### Figma template provided
![btn](https://user-images.githubusercontent.com/58692590/148635344-d7bffc0d-b828-4f45-b3c2-851b55fbd480.png)